### DISCUSSION TITLE
Somfy Zigbee 24V roller blinds motor

### DISCUSSION THREAD

**Discussion by smoothquark** [Koenkk/zigbee2mqtt#24296] [Devices]
I am planning to buy some of these blinds but for various reasons, I am planning on getting the 24v version rather than the li-ion ones. I see that the li-ion version is supported (Somfy-1241752). My dealer is checking the model no. for me but there seems to be more than one version? Am I safe to assume that the 24v version will work the same? I think the Set & Go app mentioned in the docs may not be available for newer versions of Android? At least, it does not appear in the Play Store for me but I think the Somfy Tahoma (Pro) is what I need? I don't plan to get the Tahoma Hub if I can avoid it. Thank you.

**Comment by smoothquark** [comment]
The model no. for the 24V version is 1245549.

**Comment by heisenberg2980** [comment]
My understanding is all the Somfy zigbee motors follow the same zigbee 3.0 standard. 

On a related note, I created the issue https://github.com/Koenkk/zigbee2mqtt/issues/24607 because I have been able to pair a blind with zigbee2mqtt, but the data provided by the blind is not properly mapped, so both the position and the state (open/closed) are not displayed in HA.

**Comment by smoothquark** [comment]
Thank you. The blinds are being installed at the end of the month and I will have a better idea then.

**Comment by heisenberg2980** [comment]
Just a quick tip that you won´t find in any instructions manual. When you put the blind in discovery mode by holding the button, by default the blind creates its own zigbee network, so it won´t join z2m. In order to pair the blind with z2m you must hold the button in the blind until it does 2 consecutive jogs, that means the blind is ready to join other networks and it should join z2m immediately.

**Comment by smoothquark** [comment]
Thank you. That is very helpful, indeed.
